pages/event-resources.jsx

```diff
@@ -0,0 +1,407 @@
+var React = require('react');
+var ImageTag = require('../components/imagetag.jsx');
+
+var LinksContainer = React.createClass({
+  render: function() {
+    return (
+      <div className="row">{this.props.children}</div>
+    );
+  }
+});
+var PageLinker = React.createClass({
+  render: function() {
+    return (
+      <div className="page-linker col-sm-3 col-md-3 col-lg-3">
+        <a href={this.props.href}>{this.props.head}</a>
+        {this.props.children}
+      </div>
+    );
+  }
+});
+var EventItem = React.createClass({
+  render: function() {
+    return (
+      <div className="event-item">
+        <div className="event-icon">
+          <div className="participants-number">{this.props.participants}</div>
+          <div className="participants-label">PARTICIPANTS</div>
```

Comment:
I prefer to use non-uppercase text (e.g. "Participants") and just use CSS to turn it into all-caps, but it's up to you.